net/davix/src/RRawFileDavix.cxx

```diff
@@ -94,6 +95,7 @@ void ROOT::Internal::RRawFileDavix::ReadVImpl(RIOVec *ioVec, unsigned int nReq)
    Davix::DavIOVecInput in[nReq];
    Davix::DavIOVecOuput out[nReq];
 
+   memset(in, 0, sizeof(Davix::DavIOVecInput) * nReq);
```

Comment:
Without knowing what `Davix::DavIOVecInput` is, if it is an aggregate or not, whether you care about also zeroing the padding bits or not, or whether any members for that type have non-zero default initializers that you want to override:
I got a huge fan of `{}` for initializing objects:
`Davix::DavIOVecOuput out[nReq]{};`
The syntax is much shorter and there is less potential for getting the `memset` wrong. It's also more robust against refactorings touching the definition and failing to notice the `memset` two lines below.

Reply:
Nice, thanks!  I tried with `{0}` first, which didn't work because `DavIOVecInput` is a struct.  Just `{}` does the trick.